.circleci/config.yml

```diff
@@ -0,0 +1,88 @@
+---
+version: 2.1
+
+docker_auth: &docker_auth
+  username: $DOCKERHUB_USERNAME
+  password: $DOCKERHUB_PASSWORD
+
+defaults: &defaults
+  docker:
+    - image: greenpeaceinternational/circleci-base:latest
+      auth:
+        <<: *docker_auth
+  working_directory: /home/circleci/
+jobs:
+  clone-tokens-repo:
+    <<: *defaults
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Clone planet4-design-tokens
+          command: |
+            mkdir tmp && cd "$_"
+            git clone -b PLANET-7002_implement-ci-automation https://github.com/greenpeace/planet4-design-tokens
+      - persist_to_workspace:
+          root: ./tmp/planet4-design-tokens
+          paths:
+            - ""
+  build-tokens:
+    <<: *defaults
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - attach_workspace:
+          at: ./tmp/planet4-design-tokens
+      - run:
+          name: Convert json tokens into a sass file
+          command: |
+            cd ./tmp/planet4-design-tokens
+            python3 ./bin/build.py
+      - persist_to_workspace:
+          root: ./tmp/planet4-design-tokens
+          paths:
+            - ""
+  release-tokens:
+    <<: *defaults
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - attach_workspace:
+          at: ./tmp/planet4-design-tokens
+      - run:
+          name: Create new release of tokens
+          command: |
+            python3 ./tmp/planet4-design-tokens/bin/release.py
+            mv ./tmp/planet4-design-tokens/src/_tokens.scss /home/circleci/tmp
+            ls -la /home/circleci/tmp
+      - persist_to_workspace:
+          root: ./tmp
+          paths:
+            - ""
+  promote-tokens-to-theme:
+    <<: *defaults
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - attach_workspace:
+          at: ./tmp
+      - run:
+          name: Promote tokens to master-theme
+          command: |
+            python3 ./tmp/planet4-design-tokens/bin/promote.py "./tmp"
+workflows:
+  main:
+    jobs:
+      - clone-tokens-repo
```

Comment:
I don't think a seperate job for cloning is needed sicne the repo we want is the one the pipeline is running from. Using the [builtin `checkout`](https://circleci.com/docs/configuration-reference/#checkout) command on the `build-tokens` job should work. ([for instance](https://github.com/greenpeace/planet4-master-theme/blob/main/.circleci/config.yml#L231))